docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,81 @@
+# Tutorial: Deploy the Penpot charm
```

Comment:
For my understanding, should "Penpot" be capitalised for all instances? I notice that it appears in both ways throughout the tutorial

Reply:
Thanks for catching this, will update.